### DISCUSSION TITLE
OpenSign Login page briefly displays and vanishes

### DISCUSSION THREAD

**Discussion by vmsman** [OpenSignLabs/OpenSign#1384] [Q&A]
I self hosted OpenSign in Docker on Linux.  The docker compose logs -f looks very clean and the database has been created.  In going to my public https page or accessing directly on the local address at port 3000 I can see the login page briefly and then I receive the following error.
`
Unable to connect to the OpenSign server. If you are self-hosting OpenSign, please ensure that all the steps in the documentation have been followed correctly. If you're running OpenSign locally, you might be accessing it through an incorrect port number.
`
I have closely examined the .env.prod and the docker-compose.yml and I see no issues and the logs make it look like it is running fine.  Any suggestions on what I might look at?

**Comment by apalaugonzalez** [comment]
I am having the same issue.

Update: I was using Brave when getting the error, changing to safari worked for me.

**Reply by andrew-opensignlabs** [reply]
Its already mentioned in the docs that you should access it from port 3001 and not 3000.

**Comment by vmsman** [comment]
I got closer to getting it to work on port 3000.  Port 3001 doesn't have anything running on it.  Yes, I read the documentation.  I posted because it does not seem to be functioning per the documentation.

**Reply by andrew-opensignlabs** [reply]
Can you share the exact steps you followed to install? Are all the docker containers in running state? If your setup is successful you should have it running on port 3001.

**Reply by cripepi** [reply]
@andrew-opensignlabs I am having the same issue. I tried accessing via the following URLs:
- https://localhost:3001
- http://localhost:3001
- https://localhost:3000
- http://localhost:3000

It seems to work at http://localhost:3000 only on Safari. On every other browser and the suggested https://localhost:3001, I get the same error as vmsman first mentioned ("Unable to connect..."). I've been sitting here for several hours working through the documentation, and nothing.

I'm running Ubunutu 22.04 LTS and initially tried running each script on https://docs.opensignlabs.com/docs/self-host/docker/run-locally. Neither worked.

I then added SMTP info to the .env.local_dev file (I did the same for .env.prod when it was built). Nothing.

In a last-ditch effort, I altered the port mappings within docker-compose.yml and rebuilt the container. Nothing.

Additional notes:
I initially tried running this on AWS (Ubuntu 22.04 LTS). 4 cores, 16GB RAM, 320 GB storage. When it didn't work, I thought it was some weird AWS issue. To save time, I tried running it on a local dedicated "sandbox" host (in my environment, each sandbox host is wiped at the start of a new project, so this was a clean install of everything, and the OpenSign containers are the only ones running).

**Reply by cripepi** [reply]
A more specific error from Chrome:

This site can’t provide a secure connection
xxx.xxx.xx.xx (VLAN IP) sent an invalid response.
[Try running Windows Network Diagnostics](javascript:diagnoseErrors()).
ERR_SSL_PROTOCOL_ERROR

It only works on any Apple device running Safari. You can load the login screen at http://xxx.xxx.xx.xx:3000. Weird, right? Any suggestions?

**Reply by vmsman** [reply]
I have suspected this as some type of browser issue.  I have the login screen popup and then the error message I stated initially.  I never got Opensign to work because of this.  It seems to be severely limited to very specific browser configurations.

**Reply by vmsman** [reply]
As it turns out, it cannot run from any Chrome derived browser.  I got it to run from Firefox, but there seems to be no way to register an initial account and therefore no way to login.  As a suggestion, leave caddy out of the configuration as many folks already have reverse proxy running.  My reverse proxy is NPM.

**Reply by paulo-roger** [reply]
> As it turns out, it cannot run from any Chrome derived browser. I got it to run from Firefox, but there seems to be no way to register an initial account and therefore no way to login. As a suggestion, leave caddy out of the configuration as many folks already have reverse proxy running. My reverse proxy is NPM.

Same issue here

**Comment by andrew-opensignlabs** [comment]
Can you guys share the browser console logs?
